client/boot/common.js

```diff
@@ -264,10 +282,23 @@ function setupErrorLogger( reduxStore ) {
 		errorLogger.saveExtraData( { lastTracksEvent } )
 	);
 
+	let prevPath;
 	page( '*', function ( context, next ) {
-		errorLogger.saveNewPath(
-			context.canonicalPath.replace( getSiteFragment( context.canonicalPath ), ':siteId' )
+		const path = context.canonicalPath.replace(
+			getSiteFragment( context.canonicalPath ),
+			':siteId'
 		);
+		// Also save the context to Sentry for easier debugging.
+		addBreadcrumb( {
+			category: 'navigation',
+			data: {
+				from: prevPath ?? path,
+				to: path,
+				should_capture: true, // Hint that this is our own breadcrumb, not the default navigation one.
```

Comment:
What is the practical effect of that in Sentry?

Reply:
Right, so this is actually deleted before getting sent through the API. The purpose is to basically indicate that this event is our own custom navigation event. That way, we discard any navigation breadcrumbs tracked by sentry, and only keep breadcrumbs that have this flag on them. This is accomplished with a small amount of logic in the "beforeBreadcrumb" handler.

Reply:
Ah, makes sense 👍🏻